dev-support/docker/Dockerfile

```diff
@@ -92,6 +96,21 @@ ENV MAVEN_HOME /usr
 ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64
 ENV FINDBUGS_HOME /usr
 
+#######
```

Comment:
Installing gcc-9 and this version of cmake in this way is a little hard to maintain.
I am wondering if it makes sense to just go to focal.
@aajisaka upgraded to bionic in HADOOP-16054.
I wonder if we should just upgrade to focal as it is LTS.
@ayushtkn you were also involved in that upgrade; what are your thoughts on upgrading to focal?

Reply:
Sure @goiri . I've started working on a PR to move to Focal - https://github.com/apache/hadoop/pull/2613.

Reply:
Yahh, upgrading to focal would be good

Reply:
@goiri , @aajisaka , @ayushtkn I tried to upgrade to Focal and it seems like a lot of unit tests are failing - https://github.com/apache/hadoop/pull/2613#issuecomment-761768659.
I ran all the tests on the current trunk and interestingly, there are already so many existing unit test failures - https://github.com/apache/hadoop/pull/2623#issuecomment-761741221

I would like to request that we decouple this PR from upgrading to Focal and proceed with the review of this PR.

Reply:
@goiri , @aajisaka , @ayushtkn now that the Focal upgrade PR is merged, could we please proceed with the review of this PR?